apps/backend/src/app.ts

```diff
@@ -21,6 +21,7 @@ app.route("/course/:courseID").get(getCourseByID);
 app.route("/courses").get(getCourses);
 app.route("/courses").post(isUser, getCourses);
 app.route("/courses/all").get(getAllCourses);
+app.route("/courses/relations").get(getCourseRelations);
```

Comment:
Is this endpoint being used? I don’t see it being used so if it’s not it would be best to remove the dead code